kanidm_client/src/asynchronous.rs

```diff
@@ -656,22 +657,22 @@ impl KanidmAsyncClient {
         r.map(|v| v.entries)
     }
 
-    pub async fn create(&self, entries: Vec<Entry>) -> Result<bool, ClientError> {
+    pub async fn create(&self, entries: Vec<Entry>) -> Result<(), ClientError> {
         let c = CreateRequest { entries };
         let r: Result<OperationResponse, _> = self.perform_post_request("/v1/raw/create", c).await;
-        r.map(|_| true)
+        r.map(|_| ())
     }
 
-    pub async fn modify(&self, filter: Filter, modlist: ModifyList) -> Result<bool, ClientError> {
+    pub async fn modify(&self, filter: Filter, modlist: ModifyList) -> Result<(), ClientError> {
         let mr = ModifyRequest { filter, modlist };
         let r: Result<OperationResponse, _> = self.perform_post_request("/v1/raw/modify", mr).await;
```

Comment:
It may be worth another ticket to remove OperationResponse as I think it does nothing.

Reply:
Agreed, I can open a ticket once this gets merged to avoid conflicts